boards/sensry/ganymed_bob/Kconfig.defconfig

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 sensry.io
+
+## booting
+config BOOTLOADER_MCUBOOT
+	default n
+
+## scheduling
+config TIMESLICING
+	default y
+config TIMESLICE_PER_THREAD
+	default y
+config TIMESLICE_SIZE
+	default 10
+
+## serial
+config SERIAL_SUPPORT_INTERRUPT
+	default n
+config UART_ASYNC_API
+	default n
```

Comment:
`CONFIG_UART_ASYNC_API` is disabled by default so to explicitly do so.

Reply:
fixed